IPython/core/prompts.py

```diff
@@ -350,64 +294,39 @@ def update_prompt(self, name, new_template=None):
             self.templates[name] = multiple_replace(prompt_abbreviations, new_template)
         # We count invisible characters (colour escapes) on the last line of the
         # prompt, to calculate the width for lining up subsequent prompts.
-        invis_chars = _lenlastline(self._render(name, color=True)) - \
-                        _lenlastline(self._render(name, color=False))
+        invis_chars = _lenlastline(self._render(name)) - \
+                        _lenlastline(self._render(name))
```

Comment:
If you do work on this, you should be aware of https://github.com/ipython/ipython/issues/8724.